packages/store/test/storagecoder/StorageCoder.t.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "forge-std/Test.sol";
+import { SliceLib } from "../../src/Slice.sol";
+import { EncodeArray } from "../../src/storagecoder/EncodeArray.sol";
+
+contract StorageCoderTest is Test {
```

Comment:
Should we add a few lower level tests for `StorageCoder` itself in addition to the higher level `EncodeArray` / `DecodeSlice`?

Reply:
A lot of tests are refactored and added in #396, you should review them there